widget/embedded/src/index.ts

```diff
@@ -84,9 +106,14 @@ export {
   useWallets,
   useWidgetEvents,
   customizedThemeTokens,
+  readAccountAddress,
   MainEvents,
+  Networks,
```

Comment:
I'm a bit concerned about exporting so many things from `embedded`. Some of them is Rango specific I believe and will be deprecated soon like `Networks` and `WalletTypes`.

For more explicitly separate them, I suggest to export them using a separate `export`.

```
export {
  useWallets,
  useWidgetEvents,
  customizedThemeTokens,
  ...
 }
 
 // Internal exports for Rango
 export {
 Networks,
 WalletTypes,
 ...
 }
```

 We can do same thing for `exprot type { ...}` as well.
 
 The final goal keeping these internal things separated somehow, only put them on main api (export) when we are sure they should be there.